roles/openshift_management/tasks/add_many_container_providers.yml

```diff
@@ -0,0 +1,27 @@
+---
+- hosts: "{{ groups['masters'][0] }}"
+  tasks:
+  - name: Include providers/management configuration
+    include_vars:
+      file: "{{ openshift_management_many_container_providers_config }}"
+
+  - name: Ensure this cluster is a container provider
+    uri:
+      url: "https://{{ management_server['hostname'] }}/api/providers"
+      body_format: json
+      method: POST
+      user: "{{ management_server['user'] }}"
+      password: "{{ management_server['password'] }}"
+      validate_certs: no
```

Comment:
Is this always no?

Reply:
Not always a no. This could become more complicated if we turned that on by default. OCP is going to ship self-signed certs by default for everything so I defaulted to no. That will need to be cleaned up.